### PR TITLE
Update training operators doc

### PR DESCRIPTION
**Pull request by johnugeorge**
TFJob And PyTorchJob pages are updated with latest training operator repo changes

Related: #2915 

/cc @Jeffwan

### CODE REVIEW

File: content/en/docs/components/training/tftraining.md [hunk 1]
```diff
@@ -213,19 +213,18 @@ consists of 3 fields
...
+Deploy the TFJob resource to start training:
 
 ```
-git clone https://github.com/kubeflow/tf-operator
-cd tf-operator/examples/v1/mnist_with_summaries
-# Deploy the event volume
-kubectl apply -f tfevent-volume
-# Submit the TFJob
-kubectl apply -f tf_job_mnist.yaml
+kubectl create -f simple.yaml
```
Comment (Jeffwan) [inline content/en/docs/components/training/tftraining.md:227]: Agree. It's easier to use non volume dependency jobs as example

**Review by Jeffwan — APPROVED** [review]

**Comment by Jeffwan** [comment]
Thanks @johnugeorge! 

/lgtm

/cc @RFMVasconcelos

File: content/en/docs/components/training/tftraining.md [hunk 2]
```diff
@@ -237,7 +236,7 @@ kubectl -n kubeflow get tfjob mnist -o yaml
 Delete it
```
Comment (andreyvelich) [inline content/en/docs/components/training/tftraining.md:236]: We should change `kubectl -n kubeflow get tfjob mnist -o yaml` to `kubectl -n kubeflow get tfjob tfjob-simple -o yaml`

**Review by Jeffwan — COMMENTED** [review]
I forget to mention that can we make title and description consistent between frameworks?
@johnugeorge 

image

File: content/en/docs/components/training/pytorch.md [hunk 3]
```diff
@@ -7,15 +7,12 @@ weight = 15
 
 {{% stable-status %}}
 
-This guide walks you through using PyTorch with Kubeflow.
+This page describes PyTorchJob for training a machine learning model with PyTorch.
```
Comment (andreyvelich) [inline content/en/docs/components/training/pytorch.md:10]: Should we describe what is PyTorchJob here also ?

```suggestion
This page describes PyTorchJob for training a machine learning model with PyTorch.
PyTorchJob is a Kubernetes [custom resource](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) that you can use to run distributive PyTorch training jobs on Kubernetes. The Kubeflow implementation of PyTorchJob is in [`training-operator`](https://github.com/kubeflow/tf-operator).
```

**Review by andreyvelich — COMMENTED** [review]
Thank you for updating this @johnugeorge!
Probably we should also update the [MXNet Operator](https://www.kubeflow.org/docs/components/training/mxnet/).

cc @kubeflow/wg-training-leads

**Comment by Jeffwan** [comment]
I will help  update the MXNet part. @andreyvelich

**Pull request merged**